packages/react-core/tsconfig.cjs.json

```diff
@@ -4,5 +4,11 @@
     "module": "CommonJS",
     "outDir": "dist"
   },
-  "exclude": ["node_modules", "src/**/*.test.ts", "src/**/*.test.tsx"]
+  "exclude": [
+    "node_modules",
+    "src/**/*.spec.ts",
+    "src/**/*.spec.tsx",
```

Comment:
Unrelated to this PR, but we also needed to exclude `.spec` files for building.

Reply:
Good catch